features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesFlowNode.kt

```diff
@@ -59,6 +62,7 @@ class MessagesFlowNode @AssistedInject constructor(
     @Assisted buildContext: BuildContext,
     @Assisted plugins: List<Plugin>,
     private val sendLocationEntryPoint: SendLocationEntryPoint,
+    private val viewLocationEntryPoint: ViewLocationEntryPoint,
```

Comment:
Nit: could this be `ShowLocationEntryPoint` or `LocationViewerEntryPoint`? Seeing `ViewLocation` here makes me wonder if it has something to do with view layouting for some reason.

Reply:
Yeah, `ViewLocationView` was definitely a bit weird :joy:. Done.